src/Evolution/Systems/Cce/ScriPlusValues.cpp

```diff
@@ -0,0 +1,374 @@
+       2.0 * get(boundary_r) * dy_j_at_scri *
+           (conj(eth_dy_u_at_scri) +
+            conj(eth_r_divided_by_r_view) * conj(dy_u_at_scri)) +
+       2.0 * get(boundary_r) * dy_j_at_scri * conj(dy_du_j_at_scri) -
+       2.0 * get(boundary_r) * dy_dy_w_at_scri) /
+      exp_2_beta_at_scri;
+}
+
+void CalculateScriPlusValue<Tags::ScriPlus<Tags::Psi1>>::apply(
+    const gsl::not_null<Scalar<SpinWeighted<ComplexDataVector, 1>>*> psi_1,
+    const Scalar<SpinWeighted<ComplexDataVector, 0>>& dy_dy_bondi_beta,
+    const Scalar<SpinWeighted<ComplexDataVector, 1>>& eth_dy_dy_bondi_beta,
+    const Scalar<SpinWeighted<ComplexDataVector, 2>>& dy_bondi_j,
+    const Scalar<SpinWeighted<ComplexDataVector, 1>>& dy_bondi_q,
+    const Scalar<SpinWeighted<ComplexDataVector, 1>>& dy_dy_bondi_q,
+    const Scalar<SpinWeighted<ComplexDataVector, 0>>& boundary_r,
+    const Scalar<SpinWeighted<ComplexDataVector, 1>>& eth_r_divided_by_r,
+    const size_t l_max, const size_t number_of_radial_points) noexcept {
+  const size_t number_of_angular_points =
+      Spectral::Swsh::number_of_swsh_collocation_points(l_max);
+
+  const SpinWeighted<ComplexDataVector, 0> dy_dy_beta_at_scri;
+  make_const_view(make_not_null(&dy_dy_beta_at_scri), get(dy_dy_bondi_beta),
+                  (number_of_radial_points - 1) * number_of_angular_points,
+                  number_of_angular_points);
+
+  const SpinWeighted<ComplexDataVector, 1> eth_dy_dy_beta_at_scri;
+  make_const_view(make_not_null(&eth_dy_dy_beta_at_scri),
+                  get(eth_dy_dy_bondi_beta),
+                  (number_of_radial_points - 1) * number_of_angular_points,
+                  number_of_angular_points);
+
+  const SpinWeighted<ComplexDataVector, 2> dy_j_at_scri;
+  make_const_view(make_not_null(&dy_j_at_scri), get(dy_bondi_j),
+                  (number_of_radial_points - 1) * number_of_angular_points,
+                  number_of_angular_points);
+  const SpinWeighted<ComplexDataVector, 1> dy_q_at_scri;
+  make_const_view(make_not_null(&dy_q_at_scri), get(dy_bondi_q),
+                  (number_of_radial_points - 1) * number_of_angular_points,
+                  number_of_angular_points);
+
+  const SpinWeighted<ComplexDataVector, 1> dy_dy_q_at_scri;
+  make_const_view(make_not_null(&dy_dy_q_at_scri), get(dy_dy_bondi_q),
+                  (number_of_radial_points - 1) * number_of_angular_points,
+                  number_of_angular_points);
+
+  const SpinWeighted<ComplexDataVector, 1> eth_r_divided_by_r_view;
+  make_const_view(make_not_null(&eth_r_divided_by_r_view),
+                  get(eth_r_divided_by_r),
+                  (number_of_radial_points - 1) * number_of_angular_points,
+                  number_of_angular_points);
+
+  // extra -1/sqrt(2) factor to agree with SXS tetrad normalization
+  get(*psi_1) = -0.5 * square(get(boundary_r)) *
+                (6.0 * (eth_dy_dy_beta_at_scri +
+                        eth_r_divided_by_r_view * dy_dy_beta_at_scri) -
```

Comment:
there looks like a missing factor of `boundary_r` in this term compared to the documentation

Reply:
ah, yeah, good catch -- the documentation should be R \eth R. updated.